saleor/graphql/schema.graphql

```diff
@@ -223,7 +255,9 @@ type Query {
     last: Int
   ): CollectionCountableConnection
 
-  """Look up a product by ID."""
+  """
+  Look up a product by ID. Requires one of the following permissions to include the unpublished items: MANAGE_ORDERS, MANAGE_DISCOUNTS, MANAGE_PRODUCTS.
```

Comment:
`to include the unpublished items` what does it mean? Is it ok? I thought that this perm could be required to get an access to `product`, but maybe I am missing something

Reply:
If a product is published it will be returned. If not, you need those perms to get it.